src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -319,6 +319,24 @@ sequences using different contrast enhanced images. The label is the name of the
 contrast agent. The key `ContrastBolusIngredient` MAY be also be added in the
 JSON file, with the same label.
 
+The OPTIONAL [`task-<label>`](../99-appendices/09-entities.md#task) entity can be used in order to allow tasks during structural MR acquisitions, e.g. pre-described motion paradigms such as nodding, to be described. Task labels MUST be consistent across subjects and sessions.
```

Comment:
```suggestion
The OPTIONAL [`task-<label>`](../99-appendices/09-entities.md#task) entity can be used
in order to allow tasks during structural MR acquisitions, 
for example pre-described motion paradigms such as nodding, to be described. 
Task labels MUST be consistent across subjects and sessions.
```

Reply:
just for line breaks and to remove latin abbreviation

Reply:
@effigies the "bad latin" workflow did not catch this case of abbreviation and it seems that the validation workflow is not running. 

Is this expected? Am I missing something?